x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -158,4 +187,12 @@ export function setupSavedObjects(
     type: 'api_key_pending_invalidation',
     attributesToEncrypt: new Set(['apiKeyId']),
   });
+
+  // Encrypted attributes
+  encryptedSavedObjects.registerType({
+    type: AD_HOC_RUN_SAVED_OBJECT_TYPE,
+    attributesToEncrypt: new Set(['apiKeyToUse']),
+    // attributesToIncludeInAAD: new Set(['enabled', 'start', 'duration', 'createdAt', 'spaceId', 'rule']),
+    attributesToExcludeFromAAD: new Set(['status', 'schedule']),
```

Comment:
Note that this PR merges into a feature branch and we will not plan to merge the feature branch until after https://github.com/elastic/kibana/pull/167705 is merged